non-prod/account.hcl

```diff
@@ -0,0 +1,7 @@
+# Set account-wide variables. These are automatically pulled in to configure the remote state bucket in the root
+# terragrunt.hcl configuration.
+locals {
+  account_name   = "non-prod"
+  aws_account_id = "000000000"
```

Comment:
NIT: Consider different account IDs for the various files (e.g., 1111111, 222222) just to make it a bit clearer.

Reply:
Good idea! Done: d7046c2